tests/test_file.py

```diff
@@ -382,6 +382,27 @@ def test_file_head(self):
         self._test_file_head(Replica.aws)
         self._test_file_head(Replica.gcp)
 
+    def test_drs_file_head(self):
```

Comment:
comments needed here about what the tests are doing, and why that UUID is being used; ei. "its a file within the fixture bucket"